gcp-cloud-functions/sendgrid-email-scheduled-reminder/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "sendgrid-email-scheduled-reminder",
+  "description": "Cloud Function sends an email when triggered by Cloud Task",
+  "main": "index.js",
+  "private": true,
+  "engines": {
+    "node": ">=20.0.0"
+  },
+  "dependencies": {
```

Comment:
At some point, we should look into upgrading these dependencies as well. At least NodeJS and gen 2 Cloud Functions are used now, compared to the original tutorial.